Source/WTF/wtf/Stopwatch.h

```diff
@@ -90,6 +96,23 @@ inline Seconds Stopwatch::elapsedTimeSince(MonotonicTime timeStamp) const
     return m_elapsedTime + (timeStamp - m_lastStartTime);
 }
 
+inline std::optional<Seconds> Stopwatch::fromMonotonicTime(MonotonicTime timeStamp) const
```

Comment:
I'm surprised there aren't `TestWebKitAPI` tests for Stopwatch. Otherwise I'd have expected some basic tests for this. If you're motivated enough to add them that would be awesome, but probably not a requirement =)